NuGet.config

```diff
@@ -1,8 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
-    <add key="CoreFx" value="https://dotnet.myget.org/f/dotnet-core/api/v3/index.json" />
+    <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
```

Comment:
In this repo we put the final package together by taking the OS specific packages from the feed so, no you can't get rid of this file. I did get rid of NuGet.config file in the libuv-build repo where they were not needed.